### PR TITLE
Leonidl/theming/slider - split Slider CSS and added default & Wix themes WIP

### PR DESCRIPTION
**Pull request by LeonidLevi**

### CODE REVIEW

File: src/components/slider/slider.st.css [hunk 1]
```diff
@@ -78,234 +26,85 @@
...
-.root:x .bar{
-    height: value(trackSize);
-    width: 100%;
+    position: absolute;
+    top:0;
+    left:0;
+    right:0;
+    bottom:0;
+    margin:auto;
+
```
Comment (tyv) [inline src/components/slider/slider.st.css]: empty line

File: src/components/slider/slider.st.css [hunk 2]
```diff
@@ -78,234 +26,85 @@
...
 }
 
-.root:xReverse .bar{
+.root:x .bar, .root:xReverse .bar {
     height: value(trackSize);
     width: 100%;
 }
 
-.root:y .bar{
+.root:y .bar, .root:yReverse .bar {
```
Comment (tyv) [inline src/components/slider/slider.st.css]: no space before braces

File: src/components/slider/slider.st.css [hunk 3]
```diff
@@ -78,234 +26,85 @@
...
+    top:0;
+    left:0;
+    right:0;
+    bottom:0;
+    margin:auto;
+    
 }
 
-.root:xReverse .bar{
+.root:x .bar, .root:xReverse .bar {
```
Comment (tyv) [inline src/components/slider/slider.st.css]: can we use new line each time after `,`?

File: src/components/slider/slider.st.css [hunk 4]
```diff
@@ -78,234 +26,85 @@
...
+.mark {}
 
-.root:yReverse .handle{
-    margin: value(middleOfHandle) 0px 0px value(middleOfBar);
-}
+.rangeMark {}
 
 .rangeMark,
 .mark
 {
```
Comment (tyv) [inline src/components/slider/slider.st.css]: we should use one style for curly braces: or always new line, or always last selector line

File: src/themes/default/components/slider.st.css [hunk 5]
```diff
@@ -1,66 +1,208 @@
...
+    barBorderRadius: 0px;
+    barThickness:4px;
+}
+
 Slider {
+    
+}
+
+Slider:x, Slider:xReverse {
     width:288px;
```
Comment (tyv) [inline src/themes/default/components/slider.st.css]: should something to have fixed width? 
also no space after colon

File: src/themes/default/components/slider.st.css [hunk 6]
```diff
@@ -1,66 +1,208 @@
...
+    transform: translateX(50%);
+}
+
+Slider:y::handle, 
+Slider:yReverse::handle,
+Slider:y::mark, 
+Slider:yReverse::mark, 
+Slider:y::rangeMark,
+Slider:yReverse::rangeMark {
+
```
Comment (tyv) [inline src/themes/default/components/slider.st.css]: extra line

File: src/themes/default/components/slider.st.css [hunk 7]
```diff
@@ -1,66 +1,208 @@
...
+    background-color: value(color_Disabled);
+}
+
+Slider:disabled::rangeBar {
+    background-color: transparent;
+}
+
+Slider:error::rangeMark,
+Slider:error::rangeBar
+{
```
Comment (tyv) [inline src/themes/default/components/slider.st.css]: same story with brace place

File: src/themes/wix/components/slider.st.css [hunk 8]
```diff
@@ -1,66 +1,133 @@
...
+}
+
 Slider::handle {
     height: 16px;
     width: 16px;
-    background: #FFFFFF;
-    border:none;
+    border-radius: 50%;
+    background: value(sliderHandleColor);
+    border:4px solid white;
```
Comment (tyv) [inline src/themes/wix/components/slider.st.css]: no white space before colon

File: src/themes/wix/components/slider.st.css [hunk 9]
```diff
@@ -1,66 +1,133 @@
...
     transition: box-shadow 0.3s ease-in-out;
 }
 
-Slider::handle:hover,
-Slider:hover::handle,
-Slider:active::handle {
-    box-shadow: 0 1px 2px 2px rgba(138,209,250,0.58);
+Slider:x::handle, 
+Slider:xReverse::handle {
+    top:0;
```
Comment (tyv) [inline src/themes/wix/components/slider.st.css]: no whitespace before colon

**Comment by LeonidLevi** [comment]
@tyv fixed, please re-check

**Comment by tyv** [comment]
tests are red

**Comment by Ta4i** [comment]
fixing tests

**Pull request merged**